Maybe work maybe not

UnknownUserName37 opened the pull request:
What's we have?

UnknownUserName37 committed:
Revert "Delete locators directory"

This reverts commit 6b90273c92fe8bdda9bd0e824443d12cee4bf23d.

UnknownUserName37 committed:
Revert "Delete conftest.py"

This reverts commit 76cef0ff6c43331e65d8ffa95c7c7b238837ecec.

UnknownUserName37 committed:
Revert "Delete tests directory"

This reverts commit 85a6923e99c3aa229157b8ee210592ededa6eca0.

UnknownUserName37 committed:
Revert "lol"

This reverts commit 3b245ce8ce855ea89923693f5a16c727a8c68012.

# Conflicts:
#	tests/test__equals_headers.py

UnknownUserName37 committed:
Revert "Delete conftest.py"

This reverts commit 76cef0ff6c43331e65d8ffa95c7c7b238837ecec.

UnknownUserName37 committed:
Revert "Delete stuff_shit directory"

This reverts commit e4efa632d96b666f2023c32a8d133d5e64f39e42.

UnknownUserName37 committed:
Revert "Delete pages directory"

This reverts commit 2472d360e6478fcec351daab42f1ef65cc4c4f35.

UnknownUserName37 committed:
Revert "Delete locators directory"

This reverts commit 6b90273c92fe8bdda9bd0e824443d12cee4bf23d.

UnknownUserName37 committed:
Try again.

UnknownUserName37 committed:
Try again.

UnknownUserName37 commented on conftest.py:
```diff
@@ -0,0 +1,31 @@
+#     options.add_argument('--headless')
+#     return options
+
+
```
wooooooooooooooooooooooooooooo

UnknownUserName37 committed:
Add new file with shortly test.

UnknownUserName37 committed:
Add new file with shortly test.

Change new file. Do shortly.

UnknownUserName37 committed:
Add new file with shortly test.

Change new file. Do shortly.

Return condition of native file. Make him long. Again. )

UnknownUserName37 committed:
Add new file with shortly test.

Change new file. Do shortly.

Return condition of native file. Make him long. Again. )

Add comment about ways in a top of files with tests.

UnknownUserName37 committed:
Add new file with shortly test.

Change new file. Do shortly.

Return condition of native file. Make him long. Again. )

Add comment about ways in a top of files with tests.

I' m understand my mistake.

Philechka commented on .idea/.gitignore:
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Лишний файл. Должен быть у тебя только на локалке. Убери из ПР

philipp-kochnov-stenn commented on .idea/inspectionProfiles/profiles_settings.xml:
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Лишний файл. Должен быть у тебя только на локалке. Убери из ПР

Philechka commented on .idea/make_a_deal.iml:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Лишний файл. Должен быть у тебя только на локалке. Убери из ПР

Philechka commented on .idea/misc.xml:
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Лишний файл. Должен быть у тебя только на локалке. Убери из ПР

Philechka commented on .idea/modules.xml:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Лишний файл. Должен быть у тебя только на локалке. Убери из ПР

Philechka commented on .idea/pythonProject2.iml:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Лишний файл. Должен быть у тебя только на локалке. Убери из ПР

Philechka commented on .idea/vcs.xml:
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Лишний файл. Должен быть у тебя только на локалке. Убери из ПР

Philechka commented on locators/locators.py:
```diff
@@ -0,0 +1,6 @@
+# from selenium.webdriver.common.by import By
```
Закомменченного кода не должно быть в ПР

Philechka commented on pages/base_page.py:
```diff
@@ -0,0 +1,17 @@
+# from selenium.webdriver.support.ui import WebDriverWait as Wait
```
Закомменченного кода не должно быть в ПР

UnknownUserName37 replied:
> Закомменченного кода не должно быть в ПР
если он не связан с текущим тестом, удаляю из ПР?

Philechka commented on pages/form_page.py:
```diff
@@ -0,0 +1,11 @@
+# from locators.locators import Locators
```
Закомменченного кода не должно быть в ПР

Philechka commented on stuff_shit/environment.yml:
```diff
@@ -0,0 +1,10 @@
+name : Skills-lvl_UP
```
Лишний файл. Должен быть у тебя только на локалке. Убери из ПР

Philechka commented on stuff_shit/geckodriver.log:
```diff
@@ -0,0 +1 @@
+1674495579114	geckodriver	INFO	Listening on 127.0.0.1:59948
```
Лишний файл. Должен быть у тебя только на локалке. Убери из ПР

Philechka commented on stuff_shit/test_api_ translator.py:
```diff
@@ -0,0 +1,32 @@
+import requests
```
Явно не имеет к нашему тесту отношения. Убрать из ПР

philipp-kochnov-stenn commented on stuff_shit/test_pyTest_test_env_test.py:
```diff
@@ -0,0 +1,35 @@
+import pytest
```
Явно не имеет к нашему тесту отношения. Убрать из ПР

Philechka commented on tests/on_a_future.py:
```diff
@@ -0,0 +1,9 @@
+# from locators.locators import Locators
```
Закомменченного кода не должно быть в ПР

Philechka commented on tests/on_the_future.py:
```diff
@@ -0,0 +1,9 @@
+# from locators.locators import Locators
```
Закомменченного кода не должно быть в ПР

Philechka commented on tests/test_second_variant.py:
```diff
@@ -0,0 +1,18 @@
+
+def test_eto_pizda_test(setup):
+    driver = setup
+    general_window, all_themes = driver.current_window_handle, driver.find_elements(By.XPATH, "//div[@class = 'title']")
```
Множественное присвоение это хорошо, но по отдельности код читаться будет проще

UnknownUserName37 committed:
Delete .gitignore

Not needed

UnknownUserName37 committed:
Delete profiles_settings.xml

Not needed

UnknownUserName37 committed:
Delete make_a_deal.iml

Not needed

UnknownUserName37 committed:
Delete misc.xml

Not needed

Philechka commented on tests/test_second_variant.py:
```diff
@@ -0,0 +1,18 @@
+def test_eto_pizda_test(setup):
+    driver = setup
+    general_window, all_themes = driver.current_window_handle, driver.find_elements(By.XPATH, "//div[@class = 'title']")
+    themes_list = [themes_list for themes_list in all_themes]
```
В чем смысл этой строки? Ты собрал на 8 строке список, а потом ровно его же пересобрал ровно в такой же. Почему не использовать изначальный список?

UnknownUserName37 committed:
Delete modules.xml

Not needed

UnknownUserName37 committed:
Delete pythonProject2.iml

Not needed

UnknownUserName37 committed:
Delete vcs.xml

Not needed

Philechka commented on tests/test_second_variant.py:
```diff
@@ -0,0 +1,18 @@
+    general_window, all_themes = driver.current_window_handle, driver.find_elements(By.XPATH, "//div[@class = 'title']")
+    themes_list = [themes_list for themes_list in all_themes]
+
+    for one_theme in range(len(themes_list)):
```
1. Что такое one_theme? Название этой временной переменной совершенно не читается. Переименовать. Что хранится в этой временной перменной?
2. Почему ты итерируешься по рейнджу, а потом используешь его значения как индексы что бы обращаться к themes_list и получать оттуда элемент? Почему сразу не итерироваться по themes_list и обращаться к его элементам напрямую через временную переменную one_theme?

Philechka commented on tests/test_second_variant.py:
```diff
@@ -0,0 +1,18 @@
+    themes_list = [themes_list for themes_list in all_themes]
+
+    for one_theme in range(len(themes_list)):
+        ActionChains(driver).key_down(Keys.LEFT_CONTROL).click(themes_list[one_theme]).perform()
```
Отсутствую корректные отступы между блоками кода. Прочитай про общую идеологию разделения кода по блокам по стандарту PEP8

UnknownUserName37 committed:
Delete environment.yml

если он не связан с текущим тестом, удаляю из ПР?

UnknownUserName37 reviewed (commented):
Неверный комментарий. 
Корректный: 
Not needed

UnknownUserName37 committed:
Delete geckodriver.log

Not needed

Philechka commented on tests/test__equals_headers.py:
```diff
@@ -0,0 +1,28 @@
+
+    general_window = driver.current_window_handle
+
+    ALL_THEMES = driver.find_elements(By.XPATH, "//div[@class = 'title']")
```
В питоне принято называть переменные маленькими буквами

Philechka commented on tests/test__equals_headers.py:
```diff
@@ -0,0 +1,28 @@
+    action = ActionChains(driver)
+
+    for one_theme in range(len(themes_list)):
+        text_for_assert = None
```
В питоне не надо объявлять переменные заранее, за редким исключением. У тебя не исключение

Philechka commented on tests/test__equals_headers.py:
```diff
@@ -0,0 +1,28 @@
+        handles = driver.window_handles[1]
+        driver.switch_to.window(handles)
+
+        text_for_assert = driver.find_element(By.XPATH, "//h1/i").text
```
Можно не сохранять в переменную, а сравнивать сразу с  driver.find_element(By.XPATH, "//h1/i").text

Philechka commented on tests/test__equals_headers.py:
```diff
@@ -0,0 +1,28 @@
+    general_window = driver.current_window_handle
+
+    ALL_THEMES = driver.find_elements(By.XPATH, "//div[@class = 'title']")
+    themes_list = [themes_list for themes_list in ALL_THEMES]
```
Эта строка безсмысленна

Philechka commented on tests/test__equals_headers.py:
```diff
@@ -0,0 +1,28 @@
+    ALL_THEMES = driver.find_elements(By.XPATH, "//div[@class = 'title']")
+    themes_list = [themes_list for themes_list in ALL_THEMES]
+
+    action = ActionChains(driver)
```
У тебя разовое использование этого класса. Тебе не надо сохранять его в переменную

Philechka commented on tests/test__equals_headers.py:
```diff
@@ -0,0 +1,28 @@
+
+    action = ActionChains(driver)
+
+    for one_theme in range(len(themes_list)):
```
Что такое one_theme? Название этой временной переменной совершенно не читается. Переименовать. Что хранится в этой временной перменной?
Почему ты итерируешься по рейнджу, а потом используешь его значения как индексы что бы обращаться к themes_list и получать оттуда элемент? Почему сразу не итерироваться по themes_list и обращаться к его элементам напрямую через временную переменную one_theme?

Philechka commented on tests/test__equals_headers.py:
```diff
@@ -0,0 +1,28 @@
+
+        action.key_down(Keys.LEFT_CONTROL).click(themes_list[one_theme]).perform()
+
+        handles = driver.window_handles[1]
```
Тебе не надо сохранять это в переменную

Philechka commented on tests/test__equals_headers.py:
```diff
@@ -0,0 +1,28 @@
+    for one_theme in range(len(themes_list)):
+        text_for_assert = None
+
+        action.key_down(Keys.LEFT_CONTROL).click(themes_list[one_theme]).perform()
```
В конце не хватает кей ап

Philechka replied:
Но по идее ни на что не влияет

UnknownUserName37 committed:
Delete test_api_ translator.py

Not needed

UnknownUserName37 committed:
Delete test_pyTest_test_env_test.py

Not needed

UnknownUserName37 committed:
Add new file with shortly test.

Change new file. Do shortly.

Return condition of native file. Make him long. Again. )

Add comment about ways in a top of files with tests.

I' m understand my mistake.

Fix text_for_assert.text. It's incorrect.

Uncomment. Micro_pep.

I RESTORE IT!

100% commit

New Branch for changes.

UnknownUserName37 committed:
Add new file with shortly test.

Change new file. Do shortly.

Return condition of native file. Make him long. Again. )

Add comment about ways in a top of files with tests.

I' m understand my mistake.

Fix text_for_assert.text. It's incorrect.

Uncomment. Micro_pep.

I RESTORE IT!

100% commit

New Branch for changes.

New day - new knowledge.

UnknownUserName37 committed:
Add new file with shortly test.

Change new file. Do shortly.

Return condition of native file. Make him long. Again. )

Add comment about ways in a top of files with tests.

I' m understand my mistake.

Fix text_for_assert.text. It's incorrect.

Uncomment. Micro_pep.

I RESTORE IT!

100% commit

New Branch for changes.

New day - new knowledge.

UnknownUserName37 committed:
Add new file with shortly test.

Change new file. Do shortly.

Return condition of native file. Make him long. Again. )

Add comment about ways in a top of files with tests.

I' m understand my mistake.

Fix text_for_assert.text. It's incorrect.

Uncomment. Micro_pep.

I RESTORE IT!

100% commit

New Branch for changes.

New day - new knowledge.

lol

UnknownUserName37 committed:
Add new file with shortly test.

Change new file. Do shortly.

Return condition of native file. Make him long. Again. )

Add comment about ways in a top of files with tests.

I' m understand my mistake.

Fix text_for_assert.text. It's incorrect.

Uncomment. Micro_pep.

I RESTORE IT!

100% commit

New Branch for changes.

New day - new knowledge.

lol

unlol + rebase on the right branch + edits.

UnknownUserName37 committed:
unlol + rebase on the right branch + edits.

UnknownUserName37 committed:
unlol + rebase on the right branch + edits. With a trash.

UnknownUserName37 committed:
unlol + rebase on the right branch + edits. Without a trash. Now.

UnknownUserName37 committed:
I want to have this files just on a local.

UnknownUserName37 committed:
Delete profiles_settings.xml

don't needed.

UnknownUserName37 committed:
Delete make_a_deal.iml

don't needed.

UnknownUserName37 committed:
Delete misc.xml

don't needed.

UnknownUserName37 committed:
Delete modules.xml

don't needed.

UnknownUserName37 committed:
Delete shelved.patch

don't needed.

UnknownUserName37 committed:
Delete shelved.patch

don't needed.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_27_01_2023_19_40__Changes_.xml

don't needed.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_27_01_2023_19_40__Changes_1.xml

don't needed.

UnknownUserName37 committed:
Delete shelved.patch

don't needed.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_27_01_2023_20_25__Changes_.xml

don't needed.

UnknownUserName37 committed:
Delete shelved.patch

don't needed.

UnknownUserName37 committed:
Delete shelved.patch

don't needed.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_27_01_2023_20_27__Changes_.xml

don't needed.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_27_01_2023_20_27__Changes_1.xml

don't needed.

UnknownUserName37 committed:
Delete shelved.patch

don't needed.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_28_01_2023_0_52__Changes_.xml

don't needed.

UnknownUserName37 committed:
Delete shelved.patch

don't needed.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_28_01_2023_0_53__Changes_.xml

don't needed.

UnknownUserName37 committed:
Delete vcs.xml

don't needed.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

Use this func for understanding again.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

Use this func for understanding again.

Yes. I think that I' m understanded that.

Now this commit for local.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

Use this func for understanding again.

Yes. I think that I' m understanded that.

Now this commit for local.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

Use this func for understanding again.

Yes. I think that I' m understanded that.

Now this commit for local.

And this for remote.

UnknownUserName37 committed:
Delete workspace.xml

Whow u got here?

Maby it' s too miss click.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

Use this func for understanding again.

Yes. I think that I' m understanded that.

Now this commit for local.

And this for remote.

Forgot key_up after deleting and recovery project.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

Use this func for understanding again.

Yes. I think that I' m understanded that.

Now this commit for local.

And this for remote.

Forgot key_up after deleting and recovery project.

Remote commit.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

Use this func for understanding again.

Yes. I think that I' m understanded that.

Now this commit for local.

And this for remote.

Forgot key_up after deleting and recovery project.

Remote commit.

So. I have a problem, again. Ok.

Local + history save.

Ye?

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

Use this func for understanding again.

Yes. I think that I' m understanded that.

Now this commit for local.

And this for remote.

Forgot key_up after deleting and recovery project.

Remote commit.

So. I have a problem, again. Ok.

Local + history save.

Ye?

Hmmmmm..

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

Use this func for understanding again.

Yes. I think that I' m understanded that.

Now this commit for local.

And this for remote.

Forgot key_up after deleting and recovery project.

Remote commit.

So. I have a problem, again. Ok.

Local + history save.

Ye?

Hmmmmm..

Hmmmmmmmmmmmmmmm///............................................

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

Use this func for understanding again.

Yes. I think that I' m understanded that.

Now this commit for local.

And this for remote.

Forgot key_up after deleting and recovery project.

Remote commit.

So. I have a problem, again. Ok.

Local + history save.

Ye?

Hmmmmm..

Hmmmmmmmmmmmmmmm///............................................

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

Use this func for understanding again.

Yes. I think that I' m understanded that.

Now this commit for local.

And this for remote.

Forgot key_up after deleting and recovery project.

Remote commit.

So. I have a problem, again. Ok.

Local + history save.

Ye?

Hmmmmm..

Hmmmmmmmmmmmmmmm///............................................

UnknownUserName37 committed:
Delete profiles_settings.xml

🤦‍♂

UnknownUserName37 committed:
Delete make_a_deal.iml

🤦‍♂

UnknownUserName37 committed:
Delete misc.xml

🤦‍♂

UnknownUserName37 committed:
Delete modules.xml

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_28_01_2023_1_32__Changes_.xml

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_28_01_2023_1_32__Changes_1.xml

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_28_01_2023_1_52__Changes_.xml

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Update_at_28_01_2023_1_49__Changes_.xml

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Update_at_28_01_2023_1_49__Changes_1.xml

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_.xml

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_1.xml

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_2.xml

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_3.xml

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_4.xml

🤦‍♂

UnknownUserName37 committed:
Delete vcs.xml

🤦‍♂

UnknownUserName37 committed:
Delete workspace.xml

🤦‍♂

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

Use this func for understanding again.

Yes. I think that I' m understanded that.

Now this commit for local.

And this for remote.

Forgot key_up after deleting and recovery project.

Remote commit.

So. I have a problem, again. Ok.

Local + history save.

Ye?

Hmmmmm..

Hmmmmmmmmmmmmmmm///............................................

How stop it? :D

Hmmm. I' m wont to KNOW! WGYYYYYYYYYYYDFYGVURIGBF^!&!&!&!

Rebase files.. Hmm Think about.

UnknownUserName37 committed:
I want to have this files just on a local.

hmmmmm........

I'm stupit idiot. I can't do this alll day......

Fucking trash files....

I do????

Commit just for local store.

Aaaaaaand go to remote! This file is really need there.

Try again. Local file. And switch to master branch.

Rebase complete.

Rebase on a PR branch complete. Push needed files.

Commit for equals master and PR(maybe_work....).

Use this func for understanding again.

Yes. I think that I' m understanded that.

Now this commit for local.

And this for remote.

Forgot key_up after deleting and recovery project.

Remote commit.

So. I have a problem, again. Ok.

Local + history save.

Ye?

Hmmmmm..

Hmmmmmmmmmmmmmmm///............................................

How stop it? :D

Hmmm. I' m wont to KNOW! WGYYYYYYYYYYYDFYGVURIGBF^!&!&!&!

Rebase files.. Hmm Think about.

UnknownUserName37 committed:
This is fucking ... not fucking. ok. relax. just local files.

UnknownUserName37 committed:
This is fucking ... not fucking. ok. relax. just local files.

And?

UnknownUserName37 committed:
This is fucking ... not fucking. ok. relax. just local files.

And?

UnknownUserName37 committed:
This is fucking ... not fucking. ok. relax. just local files.

And?

For remote..

UnknownUserName37 committed:
This is fucking ... not fucking. ok. relax. just local files.

And?

For remote.....

UnknownUserName37 committed:
Delete profiles_settings.xml


Checked out maybe_work_maybe_not and rebased it on master

UnknownUserName37 committed:
Delete make_a_deal.iml

🤦‍♂

UnknownUserName37 committed:
Delete misc.xml

🤦‍♂

UnknownUserName37 committed:
Delete modules.xml

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_29_01_2023_14_58__Changes_.xml

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_.xml

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_1.xml

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_2.xml

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_3.xml

🤦‍♂

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_5.xml

🤦‍♂

UnknownUserName37 committed:
Delete vcs.xml

🤦‍♂

UnknownUserName37 committed:
Delete workspace.xml

🤦‍♂

UnknownUserName37 committed:
This is fucking ... not fucking. ok. relax. just local files.

And?

For remote.....

UnknownUserName37 committed:
This is fucking ... not fucking. ok. relax. just local files.

And?

For remote..

How don' t push thisa?

UnknownUserName37 committed:
This is fucking ... not fucking. ok. relax. just local files.

And?

For remote..

How don' t push thisa?

UnknownUserName37 committed:
This is fucking ... not fucking. ok. relax. just local files.

And?

For remote..

How don' t push thisa?

And push this.

UnknownUserName37 committed:
Delete shelved.patch

hmm..

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_12.xml

LoudPass

UnknownUserName37 committed:
Delete workspace.xml

hmmmmmmmmm...

UnknownUserName37 committed:
This is fucking ... not fucking. ok. relax. just local files.

And?

For remote..

How don' t push thisa?

And push this.

И чё,типа теперь мне всё это отправлять? Мне это нахуй не нужно.

UnknownUserName37 committed:
And again.

UnknownUserName37 committed:
And again.

Ok. You are local.

UnknownUserName37 committed:
I' m not understand..

UnknownUserName37 committed:
Ссанина ебаная.

UnknownUserName37 committed:
Ссанина ебаная.

UnknownUserName37 committed:
Ссанина ебаная.

Ok. I' m finished with it.

UnknownUserName37 committed:
Ссанина ебаная.

Ok. I' m finished with it.

Commit.

UnknownUserName37 committed:
Delete profiles_settings.xml

Fuck it.

UnknownUserName37 committed:
Delete make_a_deal.iml

Fuck it.

UnknownUserName37 committed:
Delete misc.xml

Fuck it.

UnknownUserName37 committed:
Delete modules.xml

Fuck it.

UnknownUserName37 committed:
Delete shelved.patch

Fuck it.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_29_01_2023_16_10__Changes_.xml

Fuck it.

UnknownUserName37 committed:
Delete shelved.patch

Fuck it.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Update_at_29_01_2023_16_02__Changes_.xml

Fuck it.

UnknownUserName37 committed:
Delete shelved.patch

Fuck it.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Update_at_29_01_2023_16_10__Changes_.xml

Fuck it.

UnknownUserName37 committed:
Delete shelved.patch

Fuck it.

UnknownUserName37 committed:
Delete shelved.patch

Fuck it.

UnknownUserName37 committed:
Delete shelved.patch

Fuck it.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_.xml

Fuck it.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_1.xml

Fuck it.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_3.xml

Fuck it.

UnknownUserName37 committed:
Delete vcs.xml

Fuck it.

UnknownUserName37 committed:
Delete workspace.xml

Fuck it.

UnknownUserName37 committed:
And again. Just one File. I WANNA PUSH JUST ONE FUCKING FILE!!!!!!!!!!!

NOT THIS. BUT THIS SHIT WILL READY TO PUSH AFTER THIS FUCKING COMMIT!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

I DOOOOOOOOOOOOOOOOOOOOON' T NEED IT!

UnknownUserName37 committed:
JUST. ONE.

UnknownUserName37 committed:
..........and I' m again pushing 1000000 files................

UnknownUserName37 committed:
..........and I' m again pushing 1000000 files................

Fuck all this changed files.

UnknownUserName37 committed:
Ссанина ебаная.

Ok. I' m finished with it.

Commit.

UnknownUserName37 committed:
Ссанина ебаная.

Ok. I' m finished with it.

Commit just one file.

UnknownUserName37 committed:
Ссанина ебаная.

Ok. I' m finished with it.

Commit just one file. ONE.

UnknownUserName37 committed:
Delete profiles_settings.xml

🤦‍♂This is pizdec. And tupo.

UnknownUserName37 committed:
Delete make_a_deal.iml

🤦‍♂This is pizdec. And tupo.

UnknownUserName37 committed:
Delete misc.xml

🤦‍♂This is pizdec. And tupo.

UnknownUserName37 committed:
Delete modules.xml

🤦‍♂This is pizdec. And tupo.

UnknownUserName37 committed:
Delete shelved.patch

🤦‍♂This is pizdec. And tupo.

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_2.xml

🤦‍♂This is pizdec. And tupo.

UnknownUserName37 committed:
Delete vcs.xml

🤦‍♂This is pizdec. And tupo.

UnknownUserName37 committed:
Delete workspace.xml

🤦‍♂This is pizdec. And tupo.

UnknownUserName37 committed:
Ok. I' m finished with it.

Commit just one file. ONE.

UnknownUserName37 committed:
Ok. I' m finished with it.

Commit just one file. ONE.

UnknownUserName37 committed:
Ok. I' m finished with it.

Commit just one file. ONE.

And? And one.

UnknownUserName37 committed:
Ok. I' m finished with it.

Commit just one file. ONE.

And? And one.

Ok. Rebase. And forward..?

Сейчас здесь будет миллион файлов.. Но как отправить оидн7..

UnknownUserName37 committed:
Ok. I' m finished with it.

Commit just one file. ONE.

And? And one.

Ok. Rebase. And forward..?

Сейчас здесь будет миллион файлов.. Но как отправить оидн7..

One commit = one push pack.

One commit = one change.

How I can push just one file... hmmmmm...

UnknownUserName37 committed:
Ok. I' m finished with it.

Commit just one file. ONE.

And? And one.

Ok. Rebase. And forward..?

Сейчас здесь будет миллион файлов.. Но как отправить оидн7..

One commit = one push pack.

One commit = one change.

How I can push just one file... hmmmmm...

UnknownUserName37 committed:
Ok. I' m finished with it.

Commit just one file. ONE.

And? And one.

Ok. Rebase. And forward..?

Сейчас здесь будет миллион файлов.. Но как отправить оидн7..

One commit = one push pack.

One commit = one change.

How I can push just one file... hmmmmm...

UnknownUserName37 committed:
Delete profiles_settings.xml

.

UnknownUserName37 committed:
Delete make_a_deal.iml

UnknownUserName37 committed:
Delete misc.xml

.

UnknownUserName37 committed:
Delete modules.xml

UnknownUserName37 committed:
Delete shelved.patch

UnknownUserName37 committed:
Delete shelved.patch

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_29_01_2023_16_56__Changes_.xml

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_29_01_2023_16_56__Changes_1.xml

UnknownUserName37 committed:
Delete shelved.patch

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_29_01_2023_17_17__Changes_.xml

UnknownUserName37 committed:
Delete shelved.patch

UnknownUserName37 committed:
Delete shelved.patch

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_29_01_2023_18_21__Changes_.xml

UnknownUserName37 committed:
Delete Uncommitted_changes_before_Checkout_at_29_01_2023_18_21__Changes_1.xml

UnknownUserName37 committed:
Delete shelved.patch

UnknownUserName37 committed:
Delete shelved.patch

UnknownUserName37 committed:
Delete shelved.patch

UnknownUserName37 committed:
Delete shelved.patch

UnknownUserName37 committed:
Delete shelved.patch

UnknownUserName37 committed:
Delete shelved.patch

UnknownUserName37 committed:
Delete shelved.patch

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_.xml

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_1.xml

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_2.xml

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_3.xml

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_4.xml

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_5.xml

UnknownUserName37 committed:
Delete Uncommitted_changes_before_rebase__Changes_6.xml

UnknownUserName37 committed:
Delete vcs.xml

UnknownUserName37 committed:
Delete workspace.xml

UnknownUserName37 committed:
Ok. I' m finished with it.

Commit just one file. ONE.

And? And one.

Ok. Rebase. And forward..?

Сейчас здесь будет миллион файлов.. Но как отправить оидн7..

One commit = one push pack.

One commit = one change.

How I can push just one file... hmmmmm...

UnknownUserName37 committed:
Merge remote-tracking branch 'origin/maybe_work_maybe_not' into maybe_work_maybe_not

UnknownUserName37 committed:
add usless file.

UnknownUserName37 committed:
add usless file.

UnknownUserName37 committed:
Change comment.

UnknownUserName37 committed:
Change comment and delete useless file.

UnknownUserName37 committed:
Swap file contains.

UnknownUserName37 committed:
Test comments to my work. + change content of second file with native code.

UnknownUserName37 committed:
Revert "Swap file contains."

This reverts commit d31b1ea460e2157750ac1d59192342a2faa715af.

UnknownUserName37 committed:
Revert + back old version of code. With long and uncorrect, for memory)

UnknownUserName37 committed:
Delete duplicate file on_the_future.py + fix pep8 in exist on_a_future.py.

UnknownUserName37 committed:
Micro- fix for "show diff".
